jobclient/README.md

```diff
@@ -46,4 +46,13 @@ public class HostPlacementExample {
 }
 ``` 
 
+# Running the Tests
+
+The test coverage isn't very good at the moment, but here's how you run the existing unit tests:
```

Comment:
Imho, we don't need to mention the code coverage as that will change over time.

Reply:
Removed.